src/main/java/uk/gov/hmcts/darts/audio/service/impl/MediaRequestServiceImpl.java

```diff
@@ -66,8 +87,63 @@ private MediaRequestEntity saveAudioRequestToDb(HearingEntity hearingEntity, Int
         mediaRequestEntity.setRequestType(requestType);
         mediaRequestEntity.setStatus(OPEN);
         mediaRequestEntity.setAttempts(0);
+        mediaRequestEntity.setCreatedBy(requestor);
+        mediaRequestEntity.setLastModifiedBy(requestor);
 
         return mediaRequestRepository.saveAndFlush(mediaRequestEntity);
     }
 
+    @Override
+    public List<AudioRequestSummaryResult> viewAudioRequests(Integer userId, Boolean expired) {
+
+        CriteriaBuilder criteriaBuilder = entityManager.getCriteriaBuilder();
+        CriteriaQuery<AudioRequestSummaryResult> criteriaQuery = criteriaBuilder.createQuery(AudioRequestSummaryResult.class);
+
+        Root<MediaRequestEntity> mediaRequest = criteriaQuery.from(MediaRequestEntity.class);
+        Join<MediaRequestEntity, HearingEntity> hearing = mediaRequest.join(MediaRequestEntity_.hearing);
+        Join<HearingEntity, CourtCaseEntity> courtCase = hearing.join(HearingEntity_.courtCase);
+        Join<CourtCaseEntity, CourthouseEntity> courthouse = courtCase.join(CourtCaseEntity_.courthouse);
+
+        criteriaQuery.select(criteriaBuilder.construct(
+            AudioRequestSummaryResult.class,
+            mediaRequest.get(MediaRequestEntity_.id),
+            courtCase.get(CourtCaseEntity_.caseNumber),
+            courthouse.get(CourthouseEntity_.courthouseName),
+            hearing.get(HearingEntity_.hearingDate),
+            mediaRequest.get(MediaRequestEntity_.startTime),
+            mediaRequest.get(MediaRequestEntity_.endTime),
+            mediaRequest.get(MediaRequestEntity_.expiryTime),
+            mediaRequest.get(MediaRequestEntity_.status)
+        ));
+
+        ParameterExpression<UserAccountEntity> paramRequestor = criteriaBuilder.parameter(UserAccountEntity.class);
+        criteriaQuery.where(criteriaBuilder.and(
+            criteriaBuilder.equal(mediaRequest.get(MediaRequestEntity_.requestor), paramRequestor),
+            expiredPredicate(expired, criteriaBuilder, mediaRequest)
+        ));
+
+        criteriaQuery.orderBy(List.of(
+            criteriaBuilder.asc(courtCase.get(CourtCaseEntity_.caseNumber)),
+            criteriaBuilder.asc(mediaRequest.get(MediaRequestEntity_.startTime))
+        ));
+
+        TypedQuery<AudioRequestSummaryResult> query = entityManager.createQuery(criteriaQuery);
+
+        query.setParameter(paramRequestor, userAccountRepository.getReferenceById(userId));
+
+        return query.getResultList();
+    }
+
+    private Predicate expiredPredicate(Boolean expired, CriteriaBuilder criteriaBuilder,
+                                       Root<MediaRequestEntity> mediaRequest) {
+
+        final Predicate expiredPredicate;
+        if (FALSE.equals(expired)) {
```

Comment:
if you swap this logic, so test if its true then you dont need the .equals false and the double negative?

e.g.
```java
        if (expired) {
            expiredPredicate = criteriaBuilder.equal(mediaRequest.get(MediaRequestEntity_.status), EXPIRED);
        } else {
            expiredPredicate = criteriaBuilder.notEqual(mediaRequest.get(MediaRequestEntity_.status), EXPIRED);
        }
```

Reply:
Done in commit 1d02c3e3